test/binary.cpp

```diff
@@ -289,6 +289,28 @@ BITOP(bitxor, uintl, ^)
 BITOP(bitshiftl, uintl, <<)
 BITOP(bitshiftr, uintl, >>)
 
+#define UBITOP(func, T, op)                                           \
+    TEST(BinaryTests, Test_##func##_##T) {                            \
+        af_dtype ty   = (af_dtype)dtype_traits<T>::af_type;           \
+        const T vala  = 4095;                                         \
+        const T valc  = op vala;                                      \
+        const int num = 10;                                           \
+        af::array a   = af::constant(vala, num, ty);                  \
+        af::array c   = op a;                                         \
+        T *h_a        = a.host<T>();                                  \
+        T *h_c        = c.host<T>();                                  \
+        for (int i = 0; i < num; i++)                                 \
+            ASSERT_EQ(h_c[i], valc)                                   \
+                << "for values: " << h_a[i] << endl;                  \
+        af_free_host(h_a);                                            \
+        af_free_host(h_c);                                            \
```

Comment:
I think we have a macro that takes two two arrays for comparison. ASSERT_ARRAYS_EQ I think

Reply:
I took care of this in my local commit, will push soon after fixing the windows issue.